ui/admin/tests/acceptance/accounts/delete-test.js

```diff
@@ -67,17 +72,16 @@ module('Acceptance | accounts | delete', function (hooks) {
   test('can delete an account', async function (assert) {
     const accountsCount = this.server.db.accounts.length;
     await visit(urls.account);
-    await click('.rose-layout-page-actions .rose-dropdown-button-danger');
+    await click(MANAGE_DROPDOWN_SELECTOR);
+    await click(DELETE_ACTION_SELECTOR);
     assert.strictEqual(this.server.db.accounts.length, accountsCount - 1);
   });
 
   test('cannot delete an account without proper authorization', async function (assert) {
     instances.account.authorized_actions =
       instances.account.authorized_actions.filter((item) => item !== 'delete');
     await visit(urls.account);
-    assert.notOk(
-      find('.rose-layout-page-actions .rose-dropdown-button-danger'),
-    );
+    assert.notOk(find(MANAGE_DROPDOWN_SELECTOR), find(DELETE_ACTION_SELECTOR));
```

Comment:
`notOk` actually takes in only [one argument](https://qunitjs.com/api/assert/notOk/). 

If you want to check for both, I would recommend using two assertions for this and using the [dom assertions](https://github.com/mainmatter/qunit-dom/blob/master/API.md#assertdom). The dom assertions provide a more natural way of asserting the element doesn't exist versus asserting the truthiness of a found element. 

As an example:

`assert.dom(MANAGE_DROPDOWN_SELECTOR).doesNotExist();`

Reply:
Sounds good, thanks for the heads up Zed!

So in the case I implemented it; the line never ran?

Reply:
Exactly that, think that the way you wrote it works best for future scenarios handling those asserts.